test/common/policy_utils.go

```diff
@@ -69,6 +70,77 @@ func PatchPlacementRule(namespace, name string) error {
 	return err
 }
 
+// ApplyPlacement function creates Placement and PlacementBinding so that it will
+// always only match the targetCluster.
+func ApplyPlacement(namespace, policyName string) error {
+	By("Apply Placement and PlacementBinding" + namespace + "/" +
+		"placement-" + policyName + "/" + "placement-binding-" + "policyName" +
+		" with clusterSelector {name: " + ClusterNamespaceOnHub + "}")
+
+	yaml := `
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+metadata:
+  name: placement-binding-$policyName
+placementRef:
+  name: placement-$policyName
+  kind: Placement
+  apiGroup: cluster.open-cluster-management.io
+subjects:
+  - name: $policyName
+    kind: Policy
+    apiGroup: policy.open-cluster-management.io
+---
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: Placement
+metadata:
+  name: placement-$policyName
+spec:
+  predicates:
+    - requiredClusterSelector:
+        labelSelector:
+          matchExpressions:
+            - { key: name, operator: In, values: ["$clusterNamespaceOnHub"] }
```

Comment:
I think this would be more reliable by using native objects and then applying that using the client, either using the struct directly or using `map[string]interface{}`:
```suggestion
	placementBinding := policiesv1.PlacementBinding{
		ObjectMeta: metav1.ObjectMeta{
			Name: "",
		},
		PlacementRef: policiesv1.PlacementSubject{
			APIGroup: GvrPlacement.Group,
			Kind: "Placement",
			Name: "",
		},
		Subjects: []policiesv1.Subject{
			policiesv1.Subject{
				APIGroup: GvrPolicy.Group,
				Kind: "Policy",
				Name: "",
			},
		},
	}

	placement := clusterv1beta1.Placement{
		ObjectMeta: metav1.ObjectMeta{
			Name: "",
		},
		Spec: clusterv1beta1.PlacementSpec{
			Predicates: []clusterv1beta1.ClusterPredicate{
				clusterv1beta1.ClusterPredicate{
					RequiredClusterSelector: clusterv1beta1.ClusterSelector{
						LabelSelector: metav1.LabelSelector{
							MatchExpressions: []metav1.LabelSelectorRequirement{
								metav1.LabelSelectorRequirement{
									Key: "name",
									Operator: "In",
									Values: []string{""},
								}}}}}}}}
```

Reply:
great! idea!!